### PR TITLE
2 packages from puripuri2100/SATySFi-fonts-inconsolata at 3.001

### PR DESCRIPTION
**Pull request by puripuri2100**
This pull-request concerns:
-`satysfi-fonts-inconsolata.3.001`: SATySFi Font Package for inconsolata fonts
-`satysfi-fonts-inconsolata-doc.3.001`: Document of SATySFi Font Package for inconsolata fonts

---
* Homepage: https://github.com/puripuri2100/SATySFi-fonts-inconsolata
* Source repo: git+https://github.com/puripuri2100/SATySFi-fonts-inconsolata.git
* Bug tracker: https://github.com/puripuri2100/SATySFi-fonts-inconsolata/issues

---
:camel: Pull-request generated by opam-publish v2.0.2
# Automatic follow-ups
Choose follow-up actions.  Do not write anything after this section.
- [x] [ ] Add to snapshot `snapshot-develop` :: satysfi-fonts-inconsolata-doc.3.001 satysfi-fonts-inconsolata.3.001 :: satysfi-fonts-inconsolata-doc.3.001 satysfi-fonts-inconsolata.3.001
- ~~Add to snapshot `snapshot-stable-0-0-4`~~ (Inconsistent)
- [x] [x] Add to snapshot `snapshot-stable-0-0-5` :: satysfi-fonts-inconsolata-doc.3.001 satysfi-fonts-inconsolata.3.001 :: satysfi-fonts-inconsolata-doc.3.001 satysfi-fonts-inconsolata.3.001

### CODE REVIEW

File: packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+"""
+maintainer: "Naoki Kaneko <puripuri2100@gmail.com>"
+authors: "Naoki Kaneko <puripuri2100@gmail.com>"
+license: "CC0-1.0"
+homepage: "https://github.com/puripuri2100/SATySFi-fonts-inconsolata"
+bug-reports: "https://github.com/puripuri2100/SATySFi-fonts-inconsolata/issues"
+dev-repo: "git+https://github.com/puripuri2100/SATySFi-fonts-inconsolata.git"
+depends: [
+  "satysfi" {>= "0.0.4" & < "0.0.6"}
+  "satyrographos" {>= "0.0.2" & < "0.0.3"}
```
Comment (na4zagin3) [inline packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam:14]: GNU-style long options requires Satyrographos 0.0.2.6 or later.

```suggestion
  "satyrographos" {>= "0.0.2.6" & < "0.0.3"}
```

File: packages/satysfi-fonts-inconsolata/satysfi-fonts-inconsolata.3.001/opam [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+extra-source "inconsolata.zip" {
+  archive: "https://github.com/puripuri2100/inconsolata/archive/v3.001.zip"
+  checksum: [
+    "sha256=00d03451f5cb54b6a8276e788fdf2db227c432f78b3c4ecf6a3020c63ea451ab"
+    "sha512=f1c6708d22da2c8b8c935b8be2433c6bd1809330e35f185ec27a03bbdb5d7fcca4055747d1ff9cc9d0a85a442bfda7dafc94b6ead79f713aa7d95a0b1f229f9a"
+  ]
+}
+depends: [
+  "satysfi" {>= "0.0.4" & < "0.0.6"}
+  "satyrographos" {>= "0.0.2" & < "0.0.3"}
```
Comment (na4zagin3) [inline packages/satysfi-fonts-inconsolata/satysfi-fonts-inconsolata.3.001/opam:23]: GNU-style long options requires Satyrographos 0.0.2.6 or later.

```suggestion
  "satyrographos" {>= "0.0.2.6" & < "0.0.3"}
```

File: packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "satysfi-fonts-inconsolata" { = "%{version}%" }
```
Comment (na4zagin3) [inline packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam:15]: You need to add a dependency on `satysfi-dist`.

```suggestion
  "satysfi-dist"
  "satysfi-fonts-inconsolata" { = "%{version}%" }
```

**Review by na4zagin3 — CHANGES_REQUESTED** [review]
Thanks!  Can you fix the following lines?

**Comment by puripuri2100** [comment]
thank you! 👍

**Comment by na4zagin3** [comment]
Hmm, the current CI does not handle a case where a package supports SATySFi 0.0.4 but does not support Satyrographos 0.0.2.5.

Let me fix the CI.  If you want to merge this PR ASAP, you can drop support of SATySFi 0.0.4.

File: packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Document package for satysfi-fonts-inconsolata
+"""
+maintainer: "Naoki Kaneko <puripuri2100@gmail.com>"
+authors: "Naoki Kaneko <puripuri2100@gmail.com>"
+license: "CC0-1.0"
+homepage: "https://github.com/puripuri2100/SATySFi-fonts-inconsolata"
+bug-reports: "https://github.com/puripuri2100/SATySFi-fonts-inconsolata/issues"
+dev-repo: "git+https://github.com/puripuri2100/SATySFi-fonts-inconsolata.git"
+depends: [
+  "satysfi" {>= "0.0.4" & < "0.0.6"}
```
Comment (puripuri2100) [inline packages/satysfi-fonts-inconsolata-doc/satysfi-fonts-inconsolata-doc.3.001/opam:13]:
```suggestion
  "satysfi" {>= "0.0.5" & < "0.0.6"}
```

File: packages/satysfi-fonts-inconsolata/satysfi-fonts-inconsolata.3.001/opam [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+dev-repo: "git+https://github.com/puripuri2100/SATySFi-fonts-inconsolata.git"
+extra-source "inconsolata.zip" {
+  archive: "https://github.com/puripuri2100/inconsolata/archive/v3.001.zip"
+  checksum: [
+    "sha256=00d03451f5cb54b6a8276e788fdf2db227c432f78b3c4ecf6a3020c63ea451ab"
+    "sha512=f1c6708d22da2c8b8c935b8be2433c6bd1809330e35f185ec27a03bbdb5d7fcca4055747d1ff9cc9d0a85a442bfda7dafc94b6ead79f713aa7d95a0b1f229f9a"
+  ]
+}
+depends: [
+  "satysfi" {>= "0.0.4" & < "0.0.6"}
```
Comment (puripuri2100) [inline packages/satysfi-fonts-inconsolata/satysfi-fonts-inconsolata.3.001/opam:22]:
```suggestion
  "satysfi" {>= "0.0.5" & < "0.0.6"}
```

**Review by puripuri2100 — COMMENTED** [review]
I choose drop support of SATySFi 0.0.4.

**Comment by na4zagin3** [comment]
Thank you!

**Review by na4zagin3 — APPROVED** [review]

**Pull request merged**